dhis-2/build-docker-image.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-'dhis2/core'}
+IMAGE_APP_ROOT=${IMAGE_APP_ROOT:-'/usr/local/tomcat/webapps/ROOT'}
+IMAGE_USER=${IMAGE_USER:-'65534'}
+WAR_PATH=${WAR_PATH:-'dhis-2/dhis-web/dhis-web-portal/target/dhis.war'}
+UNARCHIVED_WAR_DIR=${UNARCHIVED_WAR_DIR:-'dhis2-war'}
+JIB_BUILD_FILE=${JIB_BUILD_FILE:-'jib.yaml'}
+
+downloaded_war_name='downloaded-dhis2.war'
+old_version_schema_prefix='2'
+# TODO change to https://releases.dhis2.org/v1/versions/stable.json
```

Comment:
What's the hold up? :smile:

Reply:
Just a reminder for myself, as this file is currently updated manually.